src/clib/lib/job_queue/qstat_proxy.sh

```diff
@@ -28,6 +29,13 @@ CACHE_INVALID=10
 
 QSTAT=`which qstat 2>/dev/null`
 
+QSTAT_OPTS=""
+# while the first arg starts with a - character:
+while [ "$1" != "${1#-}" ]; do
```

Comment:
Can't we just append `$QSTAT_OPTS` as is?

Reply:
This `while` loop is the parsing of the bash command line arguments, building `QSTAT_OPTS`